android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -457,7 +457,15 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     private void handleIncomingCallIntent(Intent intent) {
-        if (intent.getAction().equals(ACTION_INCOMING_CALL)) {
+        String intentAction = ACTION_NULL;
+        if (intent.getAction() instanceof String) {
```

Comment:
```suggestion
        if (intent == null || intent.getAction() == null) { return; }
```

Reply:
It may be unrelated to this comment in another PR:
https://github.com/hoxfon/react-native-twilio-programmable-voice/pull/164#issuecomment-770623529

but the condition to check that the intent is present should be the same as specified in that comment